net/grpc/gateway/docker/prereqs/Dockerfile

```diff
@@ -49,10 +51,11 @@ RUN ./scripts/init_submodules.sh
 ######################################
 # Stage 2: Copy source files and build
 ######################################
-FROM node:12.22.6-stretch AS copy-and-build
+FROM node:16-bullseye AS copy-and-build
```

Comment:
This change prevents the following error under arm64 when running the closure compiler as follows:
```
#0 159.1 /github/grpc-web/packages/grpc-web/node_modules/.bin/google-closure-compiler --js=exports.js --js=../../javascript --js=node_modules/google-closure-library --entry_point=grpc.web.Exports --externs=externs.js --dependency_mode=PRUNE --compilation_level=ADVANCED_OPTIMIZATIONS --generate_exports --export_local_property_definitions --js_output_file=index.js
#0 179.0 npm ERR! code ENOENT
#0 179.0 npm ERR! syscall access
#0 179.0 npm ERR! path /github/grpc-web/packages/grpc-web/node_modules/grpc-web.js
#0 179.1 npm ERR! errno -2
#0 179.1 npm ERR! enoent ENOENT: no such file or directory, access '/github/grpc-web/packages/grpc-web/node_modules/grpc-web.js'
#0 179.1 npm ERR! enoent This is related to npm not being able to find a file.
#0 179.1 npm ERR! enoent 
```

Reply:
Aha thanks for the context!

Do you mind replacing all `12.22.6` references in this repo to the same node version tho (there are [3 files](https://cs.github.com/grpc/grpc-web?q=12.22.6) afaict)?

Ideally to a more specific dot release to ensure our test [here](https://github.com/grpc/grpc-web/blob/1779661dded5b918cdd84a216f708b93ebf2384c/packages/grpc-web/docker/jsunit-test/Dockerfile#L3-L4) will always be running the same node version as the node server.